Dockerfile

```diff
@@ -23,6 +23,12 @@ RUN CGO_ENABLED=0 go build -v -o bin/eupf ./cmd/
 FROM alpine:3.18 AS runtime
 LABEL org.opencontainers.image.source="https://github.com/edgecomllc/eupf"
 
+RUN apk update \
+    # fix CVE-2023-5363, CVE-2023-5678
```

Comment:
Let's add comment to update apline image instead after https://gitlab.alpinelinux.org/alpine/aports/-/issues/15498 fix